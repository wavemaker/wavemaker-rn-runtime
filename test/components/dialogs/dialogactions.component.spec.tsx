import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmDialogactions from '@wavemaker/app-rn-runtime/components/dialogs/dialogactions/dialogactions.component';

describe('Test Dialogactions component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmDialogactions name="test_Dialogactions"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});