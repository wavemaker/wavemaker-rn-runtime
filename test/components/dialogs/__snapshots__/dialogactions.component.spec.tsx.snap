// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialogactions component should not show component when show is false 1`] = `
<View
  style={
    {
      "height": 0,
      "transform": [
        {
          "scale": 0,
        },
      ],
      "width": 0,
    }
  }
>
  <Text>
    Test children component
  </Text>
</View>
`;

exports[`Dialogactions component should render component 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "borderColor": "#cccccc",
      "borderStyle": "solid",
      "flexDirection": "row",
      "justifyContent": "flex-end",
    }
  }
/>
`;

exports[`Dialogactions component should render component with children 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "borderColor": "#cccccc",
      "borderStyle": "solid",
      "flexDirection": "row",
      "justifyContent": "flex-end",
    }
  }
>
  <Text>
    Test children component
  </Text>
</View>
`;

exports[`Dialogactions component should render component with custom style 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "borderColor": "#404040",
      "borderStyle": "solid",
      "flexDirection": "row",
      "justifyContent": "flex-end",
    }
  }
>
  <Text>
    Test children component
  </Text>
</View>
`;
